s3-bucket.tf

```diff
@@ -124,3 +124,14 @@ resource "aws_s3_bucket_policy" "remote_state_backend" {
   bucket = aws_s3_bucket.remote_state_backend.id
   policy = data.aws_iam_policy_document.encrypted_transit_bucket_policy.json
 }
+
+resource "aws_s3_bucket_public_access_block" "this" {
```

Comment:
Is this 4.x compatible?

Reply:
No - the whole module is not, I believe.

Reply:
Would it be 4.x compatible if it went into the main resource block?